dashboard/app/controllers/courses_controller.rb

```diff
@@ -30,11 +30,6 @@ def new
   end
 
   def show
-    if !params[:section_id] && current_user&.last_section_id
```

Comment:
Can we move this removal to the other blocked PR? I don't want to change anything for the current course overview page